storage/rocksdb/clone/common.cc

```diff
@@ -235,6 +235,16 @@ void fixup_on_startup() {
   if (path_exists(current_datadir_in_progress_marker_path))
     rdb_fatal_error("In-progress clone marker found in the MyRocks datadir");
 
+  // Enable sst file checksum verification if
+  // CHECKSUMS_WRITE_AND_VERIFY_ON_CLONE flag is enabled and this restart is
+  // after a clone.
+  if (myrocks::rocksdb_file_checksums ==
+          myrocks::file_checksums_type::CHECKSUMS_WRITE_AND_VERIFY_ON_CLONE &&
+      temp_dir_exists_abort_if_not_dir(in_place_temp_datadir)) {
```

Comment:
Thank you, this is a better location IMHO.I I'd also remove `temp_dir_exists_abort_if_not_dir` check for clarity, but up to you

Reply:
If ```temp_dir_exists_abort_if_not_dir``` is removed, will the restart happens on every mysqld restart, not just the restart right after clone? The intended behavior is the checksum verification only happens for clone.